geom/src/bounds.rs

```diff
@@ -92,6 +94,7 @@ impl Bounds {
     }
 
     /// Converts the boundary to the format used by `aabb_quadtree`.
+    // TODO On its way out
```

Comment:
There's a circular dependency with osm2streets. I'll merge this PR and sort it out.